### PR TITLE
Add Bloomberg's goldpinger

### PR DESCRIPTION
**Pull request by laverya**
https://github.com/bloomberg/goldpinger

### CODE REVIEW

File: addons/goldpinger/template/values.yaml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+affinity: {}
+
+## Enable this if pod security policy enabled in your cluster
+## It will bind ServiceAccount with unrestricted podSecurityPolicy
+## Ref: https://kubernetes.io/docs/concepts/policy/pod-security-policy/
+podSecurityPolicy:
+  enabled: false
+  policyName: unrestricted-psp
+
+serviceMonitor:
```
Comment (DanStough) [inline addons/goldpinger/template/values.yaml:96]: Might agree with @emosbaugh to turn this on by default, but wouldn't let it hold up the PR.

**Review by DanStough — CHANGES_REQUESTED** [review]
I think you may need to add the addon to the distro file(s), like this one for kubeadm https://github.com/replicatedhq/kURL/blob/bb9d9c3d72e9052d85b8ae8313cb6d10266a106d/scripts/distro/kubeadm/distro.sh#L35-L58

File: addons/goldpinger/template/values.yaml (hunk 1 above)
Comment (emosbaugh) [reply addons/goldpinger/template/values.yaml:96]: I would turn this on IF the prom addon is enabled

Comment (laverya) [reply addons/goldpinger/template/values.yaml:96]: Will do

Comment (laverya) [reply addons/goldpinger/template/values.yaml:96]: enabled (but not through the helm chart)

**Review by DanStough — APPROVED** [review]

**Pull request merged**